packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -31,6 +31,39 @@ typedef PaintRangeValueIndicator = void Function(PaintingContext context, Offset
 ///
 /// Used to select a range from a range of values.
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold}
+///
+/// This is what it looks like when run:
+///
+/// ![A range slider widget, consisting of 5 divisions and showing the default
+/// value indicator.](https://flutter.github.io/assets-for-api-docs/assets/material/range_slider.png)
+///
+/// This sample shows creation of a [RangeSlider] widget with 5 divisions.
+///
+/// ```dart
+/// RangeValues _values = const RangeValues(40, 80);
```

Comment:
This might be slightly clearer if it were called _currentRange

Reply:
I changed it to _currentRangeValues to keep consistent with Slider demo